spec/services/globus_cleanup_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+RSpec.describe GlobusCleanup do
+  let(:user) { create(:user, sunet_id: 'ima_user') }
+
+  let!(:job_run) { create(:job_run, :preassembly) }
+  let!(:job_run_complete) { create(:job_run, :preassembly, :accessioning_complete) }
+
+  describe '.run' do
```

Comment:
Should also test when an error is raised.

Reply:
Added a test for this